ERROR    logging:test_regist.py:46 测试编号4,测试用例标题:确认密码为空,执行失败,实际结果为:{'code': 1004, 'msg': '两次密码输入不一致'}!
ERROR    logging:test_regist.py:47 assert {'code': 1003...': '确认密码不能为空'} == {'code': 1004...: '两次密码输入不一致'}
  Differing items:
  {'msg': '确认密码不能为空'} != {'msg': '两次密码输入不一致'}
  {'code': 1003} != {'code': 1004}
  Full diff:
  - {'code': 1004, 'msg': '两次密码输入不一致'}
  ?             ^          ^^  -- ^^
  + {'code': 1003, 'msg': '确认密码不能为空'}...
  
  ...Full output truncated (2 lines hidden), use '-vv' to show
Traceback (most recent call last):
  File "C:\Program Files (x86)\PythonWorkSpace\autotest03\testcases\test_my_interface01\test_regist.py", line 43, in test_register
    assert eval(case.expect) == res_body
AssertionError: assert {'code': 1003...': '确认密码不能为空'} == {'code': 1004...: '两次密码输入不一致'}
  Differing items:
  {'msg': '确认密码不能为空'} != {'msg': '两次密码输入不一致'}
  {'code': 1003} != {'code': 1004}
  Full diff:
  - {'code': 1004, 'msg': '两次密码输入不一致'}
  ?             ^          ^^  -- ^^
  + {'code': 1003, 'msg': '确认密码不能为空'}...
  
  ...Full output truncated (2 lines hidden), use '-vv' to show